scripts/install-windows-blas-lapack.ps1

```
@@ -0,0 +1,200 @@
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$VcpkgRoot,
+
+  [string]$VcpkgRef = "master",
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'VcpkgRef' has been declared but not used.

(PSReviewUnusedParameter)